Sources/DistributedActors/ActorSystem.swift

```diff
@@ -100,20 +100,37 @@ public final class ActorSystem: _Distributed.ActorTransport, @unchecked Sendable
     // ==== ----------------------------------------------------------------------------------------------------------------
     // MARK: Metrics
 
-    // TODO: Use ManagedAtomicLazyReference to store this
-    private lazy var _metrics: ActorSystemMetrics = ActorSystemMetrics(self.settings.metrics)
-    internal var metrics: ActorSystemMetrics {
-        self._metrics
-    }
+    internal let metrics: ActorSystemMetrics
 
     // ==== ----------------------------------------------------------------------------------------------------------------
     // MARK: Cluster
+    internal class Box<T> {
+        var value: T
+        init(_ value: T) {
+            self.value = value
+        }
+    }
```

Comment:
Please tell me if this would be better at global scope on some utils.swift file for instance